webapp/sources/rudder/rudder-web/src/main/webapp/templates-hidden/components/ComponentDirectiveEditForm.html

```diff
@@ -41,27 +41,6 @@
       var hasWriteRights = true;
     </script>
   </lift:authz>
-  <script>
-  var baseUrl = contextPath ? contextPath : "/rudder";
-  var apiPath = baseUrl + '/secure/api/sharedfile';
-
-  var container = document.getElementById("filemanager-editor")
-  var fm = Elm.FileManager.init({
-      node: container,
-      flags: {
-        api: apiPath,
-        thumbnailsUrl: apiPath,
-        downloadsUrl: apiPath,
-        dir: "/",
-        hasWriteRights : hasWriteRights,
-        initRun : true;
```

Comment:
This is a syntax error so the app was not even functioning in the Directives page...
I removed the whole script because the app to keep the same behavior, it looks like it does nothing at all in this page